UserDetails/views.py

```diff
@@ -53,25 +54,28 @@ def post(self, request, *args, **kwargs):
         return self.render_to_response(context)
 
 
-class DiningHistoryView(View):
+class DiningJoinHistoryView(View, PageListView):
```

Comment:
> I understand your view on seperation. I'm a bit indifferent right now. Is there a particular bit of code that bothers you in this situation?

Nothing is bothering me about that, it was just something that I had in my head and thought maybe it's also helpful for you to have in your head while developing, it can maybe help you with design decisions :smile:. Or just ignore it, then you'll at least know why I might have done something in a particular way.

Also the rest of the comments are also just things that I notice, but they don't bother me, as you'll have seen by that I already accepted the original PR. You didn't have to take them into account, but it's extra nice that you actually did :smile: